### PR TITLE
Graal Cloud Native Launcher extension renamed.

### PR DESCRIPTION
**Pull request by dbalek**
The `Graal Cloud Native Launcher` extension has been recently renamed to `Graal Development Kit for Micronaut Launcher`

### CODE REVIEW

**Comment by jdneo** [comment]
I raised another PR https://github.com/microsoft/vscode-java-dependency/pull/850 to fix the testing issues. There are a lot of stuffs need to be update/change. Jesus...

**Comment by dbalek** [comment]
I'm waiting for an approval from my company to sign the CLA :-(

**Comment by jdneo** [comment]
Wired, but the `license/cla` has a green mark now

**Comment by jdneo** [comment]
Oh, is that because I added commit to the PR as well?

@dbalek Would you mind rebase this PR on the main branch, and remove the commit that I appended?

**Review by jdneo — APPROVED** [review]

**Pull request merged**
